iceoryx_hoofs/test/moduletests/test_lockfree_queue.cpp

```diff
@@ -41,8 +42,11 @@ struct Integer
     int value{0};
 
     // so that it behaves like an int for comparison purposes
+    // NOLINTNEXTLINE(hicpp-explicit-conversions) required for typed tests
     operator int() const
     {
+        /// @todo suppression does not work but it is a false positive
+        // NOLINTNEXTLINE(clang-analyzer-core.unitialized.UndefReturn) false positive - defaults to 0
```

Comment:
@elBoberido 

This suppression does not quite work and the warning is false positive or at least not clear to me. The return value should be well defined if the object is constructed (and we do not call it before).